jdaviz/configs/default/tests/test_aida.py

```diff
@@ -52,7 +55,8 @@ def test_set_viewport_sky(imviz_helper, image_hdu_wcs):
     # change only the center:
     new_viewport_settings = dict(
         center=SkyCoord(ra=337.5, dec=-20.8, unit='deg'),
-        fov=0.01 * u.deg
+        fov=0.01 * u.deg,
+        rotation=90 * u.deg
```

Comment:
Let's drop the rotation change from this part of the test, and add a separate (set?) of `assert`s to check rotation.

Reply:
added a new test that just covers rotation